File: src/main/java/team4/footwithme/vote/service/response/VoteItemResponse.java
package team4.footwithme.vote.service.response;

import java.util.List;

public record VoteItemResponse(
    Long voteItemId,
    String content,
    List<Long> memberIds
) {
    public static VoteItemResponse of(Long voteItemId, String contents, List<Long> memberIds) {
        return new VoteItemResponse(
            voteItemId,
            contents,
            memberIds
        );
    }
}

File: src/main/java/team4/footwithme/vote/service/response/VoteItemResponse.kt
package team4.footwithme.vote.service.response

@JvmRecord
data class VoteItemResponse(
    val voteItemId: Long?,
    val content: String?,
    val memberIds: List<Long?>?
) {
    companion object {
        fun of(voteItemId: Long?, contents: String?, memberIds: List<Long?>?): VoteItemResponse {
            return VoteItemResponse(
                voteItemId,
                contents,
                memberIds
            )
        }
    }
}
